index.ts

```diff
@@ -387,7 +387,7 @@ commander
                 nonIgnoredFilePaths.map((filePath: string) => {
                   console.error(`└─ ${filePath}`);
                 });
-                process.exit(-1);
+                process.exit(1);
```

Comment:
-1 is actually out of range for status codes, and really returns 255. Changed to 1 for consistency.